homeassistant/components/frontend/strings.json

```
@@ -23,13 +15,13 @@
     "set_theme": {
       "description": "Sets the default theme Home Assistant uses. Can be overridden by a user.",
       "fields": {
-        "mode": {
-          "description": "Theme mode.",
-          "name": "Mode"
-        },
         "name": {
-          "description": "Name of a theme.",
+          "description": "Name of the theme to use when a light-mode client requests the default theme.",
           "name": "Theme"
+        },
+        "name_dark": {
+          "description": "Name of the theme to use when a dark-mode client requests the default theme.",
+          "name": "Dark Theme"
```

This should be sentence-cased as most languages translate "theme".

```suggestion
          "name": "Dark theme"
```